shared/middleware/files_api.rb

```diff
@@ -476,6 +476,7 @@ def files_put_file(encrypted_channel_id, filename, body)
 
     bucket = FileBucket.new
     manifest = get_manifest(bucket, encrypted_channel_id)
+    manifest_is_unchanged = true
```

Comment:
Neither `copy_file` nor `put_file` have any knowledge of the manifest; they are helpers used by all the APIs in this file: source, assets, animations and files.  Only the "files API" knows about the manifest.